### DISCUSSION TITLE
FRAMEINFO.md - Video duration tolerances

### DISCUSSION THREAD

**Discussion by jkind73** [DirtBagXon/hypseus_singe_data#6] [Q&A] [answered]
According the FRAMEINFO.md file in the DL2e repository the video duration should be 1591089. I've been downloading more than 5 different version all clamming to be the one to be used. But the video duration is always 1591090.
Not sure if this represents a real problem or not, but which one is the correct value?

mediainfo --Inform='Video;%FrameCount%' dl2e.m2v
38148

mediainfo --Inform='Video;%Duration%' dl2e.m2v 
1591089

**Comment by DirtBagXon** [comment, marked as answer]
Hiya,

This is my current version:
```
mediainfo --Inform='Video;%FrameCount%' dl2e.m2v 
38148
mediainfo --Inform='Video;%Duration%' dl2e.m2v 
1591089
```

If I understand what you are asking: Is `1591090` rather that `1591089` ok  for the duration ?

You are talking about `1 millisecond` of difference in the video length, I think that can be put down to `ffmpeg` tolerances 👍🏻 

The `FrameCount` is the more important figure, but even this may have tolerances. 
Provided there are not huge differences (_10+ as a finger in the air_), you can pretty much guarantee it is the same video source.
